code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -188,7 +155,16 @@
 				reagents.reaction(target, INGEST)
 			spawn(5)
 				target.add_fingerprint(user)
-				var/trans = src.reagents.trans_to(target, amount_per_transfer_from_this)
+				var/datum/reagent/blood/B
+				for(var/datum/reagent/blood/d in src.reagents.reagent_list)

```

Comment:
Perhaps change this to ```var/datum/reagent/blood/B = locate() in src.reagents.reagent_list```
it seems silly to write a for loop that will only run one iteration.